mikeio/dataarray.py

```diff
@@ -2245,6 +2245,13 @@ def to_xarray(self):
             coords["element"] = xr.DataArray(
                 data=self.geometry.element_ids, dims="element"
             )
+        elif isinstance(self.geometry, GeometryPoint2D):
+            coords["x"] = self.geometry.x
```

Comment:
Any attributes to add from `geometry`, e.g., info on projection and units? Actually same for the other geometries here. But `GeometryPoint2D.projection` does not seem to be defined.